http-servers/src/main/java/com/palantir/remoting/http/server/WebPreconditions.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
```

Comment:
I'm not sure this is the right way to propagate the original copyright? Could you check with someone who knows?